test cases/fortran/7 generated/meson.build

```diff
@@ -1,18 +1,19 @@
 # Tests whether fortran sources files created during configuration are properly
 # scanned for dependency information
 
-project('generated', 'fortran')
+project('generated', 'fortran',
+  default_options : ['default_library=static'])
 
 conf_data = configuration_data()
 conf_data.set('ONE', 1)
 conf_data.set('TWO', 2)
 conf_data.set('THREE', 3)
 
 outfile = configure_file(
-      input : 'mod3.fpp', output : 'mod3.f90', configuration : conf_data)
+  input : 'mod3.fpp', output : 'mod3.f90', configuration : conf_data)
```

Comment:
Spurious whitespace? We use a single tab for function call continuation elsewhere.